### ISSUE TITLE
bug: FilePicker.pickMedia() having trouble with heavy videos (+20MB)

### ISSUE THREAD

**Issue by maximilien0405** [capawesome-team/capacitor-plugins#116]
### Plugin(s)

- [ ] Android Foreground Service
- [ ] Android Battery Optimization
- [ ] App Update
- [ ] Background Task
- [ ] Badge
- [ ] Cloudinary
- [ ] Datetime Picker
- [ ] File Opener
- [X] File Picker
- [ ] Managed Configurations
- [ ] NFC
- [ ] Photo Editor
- [ ] Screen Orientation

### Version

5.1.1

### Platform(s)

- [ ] Android
- [X] iOS
- [ ] Web

### Current behavior

Hi, so first of all thanks for making this plugin and providing it for free to everyone.

I'm having trouble with the File Picker plugin. In my app, a user can select images/videos from it's gallery. To do that i use this piece of code : 

`const mediasResult = await FilePicker.pickMedia({
     multiple: true,
     readData: false,
     skipTranscoding: true
})`

After that, i dismiss the modal, and opens another modal with a preview of the selected images/videos. Hovever, when i try to select a heavy video (+20mb), it dosen't work or it takes too much time to take the media..

Here is below the behavior on my device : 

https://github.com/capawesome-team/capacitor-plugins/assets/46009279/dc88ad1d-262f-4089-ae8b-aaa58f3ddf35

In the video i try to get a 8.4Mb (HEVC), a 48MB (HEVC) and a 67MB (H.264) videos. As you can see, even if i have "skipTranscoding" and "readData" to false, the heavyer the video is, the longer it takes to take the media. With the 67Mb video it dosen't even work ! (maybe i havent waited long enough)

### Expected behavior

After selecting the medias, it is supposed to close the modal, and open another modal **directly after** with the medias selected, so that the user can preview the images/videos selected. It will then upload the medias to my backend server using the "path" in the media object.

I cannot have the user wait 30 seconds to display his video, do anyone think there is a solution ? Thanks in advance !

### Reproduction

.

### Steps to reproduce

.

### Capacitor doctor

💊   Capacitor Doctor  💊 

Latest Dependencies:

  @capacitor/cli: 5.6.0
  @capacitor/core: 5.6.0
  @capacitor/android: 5.6.0
  @capacitor/ios: 5.6.0

Installed Dependencies:

  @capacitor/cli: 5.4.2
  @capacitor/core: 5.4.2
  @capacitor/android: 5.4.2
  @capacitor/ios: 5.4.2

[success] iOS looking great! 👌
[success] Android looking great! 👌

### Before submitting

- [X] I have read and followed the [bug report guidelines](https://capawesome.io/contributing/bug-reports/).
- [X] I have attached links to possibly related issues and discussions.
- [X] I understand that incomplete issues (e.g. without reproduction) are closed.

**Comment by robingenz** [comment]
Thank you for your request.

> As you can see, even if i have "skipTranscoding" and "readData" to false, the heavyer the video is, the longer it takes to take the media.

You should definitely set `skipTranscoding` to `true` as in your code above. I just want to make sure you've tried that.

**Comment by maximilien0405** [comment]
Yes skipTranscoding is set to true (sorry i miss wrote, i've fixed my typo error) :)

**Comment by robingenz** [comment]
Okay, i will look into this. But I'm a bit busy this week.

**Comment by maximilien0405** [comment]
Thank you so much ! If you need any more information don't hesitate

**Comment by robingenz** [comment]
Please provide some sample files for testing.
